From eb5ad279e166bccf30ef5c5ce402df4c17b8f1bb Mon Sep 17 00:00:00 2001
From: Modestas Vainius <modax@debian.org>
Date: Mon, 3 Jan 2011 15:03:20 +0200
Subject: [PATCH 2/2] Fix setting of trace log level via --log-level.

TRACE_LEVEL is 0 so if (!level) can't be used here.
---
 src/main.cc |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/src/main.cc b/src/main.cc
index 03af0a8..74af00f 100644
--- a/src/main.cc
+++ b/src/main.cc
@@ -443,8 +443,7 @@ namespace
       log_levels.find(level_name);
     if(found != log_levels.end())
       level = found->second;
-
-    if(!level)
+    else
       {
 	// ForTranslators: both the translated and the untranslated
 	// log level names are accepted here.
-- 
1.7.2.3

